pkg/html/releasehtml/top_level_indicators.go

```diff
@@ -18,11 +18,13 @@ func topLevelIndicators(report, reportPrev sippyprocessingv1.TestReport, release
 			<th title="How often we get to the point of running the installer.  This is judged by whether a kube-apiserver is available, it's not perfect, but it's very close." class="text-center {{ .infraColor }}">Infrastructure</th>
 			<th title="How often the install completes successfully." class="text-center {{ .installColor }}"><a href="/install?release={{ .release }}">Install</a></th>
 			<th title="How often an upgrade that is started completes successfully." class="text-center {{ .upgradeColor }}"><a href="/upgrade?release={{ .release }}">Upgrade</a></th>
+			<th title="How often CI job runs finish with all operators healthy." class="text-center {{ .finalColor }}"><a href="/operator-health?release={{ .release }}">Operator Health</a></th>
```

Comment:
is this really a critical top level indicator?

i think it's a good thing to aim for, but it doesn't in and of itself tell us anything about our CI stability or ability to ship the product.

Reply:
> is this really a critical top level indicator?
> 
> i think it's a good thing to aim for, but it doesn't in and of itself tell us anything about our CI stability or ability to ship the product.

I'm about done in 4.6, so I'm probably willing to comment out these lines and let the 4.7 TRT decide.  However, when you look at the differences between install/upgrade and operator health, you can see cases where we are able to install without issue, but cannot keep a cluster healthy for the duration of an e2e test.  Stand out cases in comparison include

1. azure - healthy less than 90% of the time, but installs over 90%.  Monitoring is particularly prone to failure
2. ovirt - installs more commonly than it ends healthy. authentication and console are particularly bad, which suggests that ingress goes offline at some point